### DISCUSSION TITLE
gssproxy and auditd timeouts

### DISCUSSION THREAD

**Discussion by pbrezina** [bootc-dev/bootc#968] [Q&A] [answered]
I have an image `quay.io/pbrezina/fedora-bootc-workstation:latest` based on fedora-bootc with workstation environment installed.

It boots fine, but gssproxy.service and auditd.service always times out with

```
Can't open PID file /run/gssproxy.pid
Can't open PID file /run/auditd.pid
```

**Comment by vrothberg** [comment, marked as answer]
Thanks for reaching out, @pbrezina !  See https://gitlab.com/fedora/bootc/base-images/-/issues/28 which will fix the issue.

As a workaround you can add the following to your Containerfile:
```Dockerfile
RUN rm -rf /var/run && ln -s /run /var/
```

**Reply by pbrezina** [reply]
Shouldn't it be `ln -s /run /var/run`?

**Reply by cgwalters** [reply]
That's the same thing, most of coreutils like `cp` and `ln` automatically add the source basename if the target is an extant directory.
